### ISSUE TITLE
Messages: `aria-label` not included in close button attributes

### ISSUE THREAD

**Issue by zaguiini** [primefaces/primereact#7364]
### Describe the bug

There is no `aria-label` attribute for screen readers. So it doesn't say "Close" out loud.

![image](https://github.com/user-attachments/assets/5366480b-a2e9-4573-ac69-cbbb82668eb5)

### Reproducer

https://stackblitz.com/edit/vitejs-vite-9jag2b?file=src%2FApp.tsx

### System Information

```Shell
System:
    OS: macOS 15.0.1
    CPU: (12) arm64 Apple M3 Pro
    Memory: 62.72 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.17.0 - ~/.nvm/versions/node/v20.17.0/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v20.17.0/bin/yarn
    npm: 10.8.2 - ~/.nvm/versions/node/v20.17.0/bin/npm
  Browsers:
    Safari: 18.0.1
```

### Steps to reproduce the behavior

Inspect the close button. It should have the "Name" property (which is the `aria-label`) attribute. But it's empty.

### Expected behavior

The "Name" property in the inspector has "Close" as its value, and the `aria-label` attribute is present.

![image](https://github.com/user-attachments/assets/d017aa1c-9bc6-40cd-86af-0177d0ec1861)
